### PR TITLE
refactor: add path file required project id and file path

### PR DESCRIPTION
**Pull request by GabyGV-GGV**
## Related Tickets & US
- [CS-43-18](https://app.clickup.com/t/868b5427h)

## Description
The endpoint should receive the project ID and the project path in order to return the adequate coverage file.

GET coverage/project/{projectId}/file?path=src/org/example/AppTest 

Also, update the [documentation](https://github.com/SoftDevSix/ArgosApplication/wiki/Endpoints) of the endpoint.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

- Run a project in file manager and get the project id.

- Enter the file path and project id example: http://localhost:8081/coverage/project/06781212-69c1-46d2-b73b-2f69d83e1edc/?filePath=test-jarasm%2Fsrc%2Fmain%2Fjava%2Forg%2Fexample%2FMultiplication.java .

- and see results.

image

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have checked my code and corrected any misspellings

- - -
This change is [Reviewable](https://reviewable.io/reviews/SoftDevSix/ArgosApplication/39)

### CODE REVIEW

File: src/main/java/com/softdevsix/presentation/controllers/FileCoverageController.java [hunk 1]
```diff
@@ -1,25 +1,40 @@
...
-    public FileCoverageController(IFileService fileService) {
+    public FileCoverageController(IFileService fileService, IProjectService projectService) {
         this.fileService = fileService;
+        this.projectService = projectService;
     }
 
-    @GetMapping("/{idFile}")
-    public ResponseEntity<FileCoverageDto> getFileCoverage(@PathVariable UUID idFile) {
-        File file = fileService.getFileById(idFile);
+    /**
```
Comment (sebastian-barra-jalafund) [inline src/main/java/com/softdevsix/presentation/controllers/FileCoverageController.java:24]: is it necesary to keep this comment?

**Review by ronaldo-mendoza-jalafund — APPROVED** [review]
> Reviewed 8 of 8 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/SoftDevSix/ArgosApplication/39)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @sebastian-barra-jalafund and @zamoranoalicia)

**Review by ronaldo-mendoza-jalafund — APPROVED** [review]
This US is approved because it meets the exit criteria.

**Review by sebastian-barra-jalafund — APPROVED** [review]
well done

File: src/main/java/com/softdevsix/presentation/controllers/FileCoverageController.java (hunk 1 above)
Comment (GabyGV-GGV) [reply src/main/java/com/softdevsix/presentation/controllers/FileCoverageController.java:24]: yes, i need this comment.

**Pull request merged**